packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx

```diff
@@ -9,7 +9,14 @@ export const MuiThemeDecorator: DecoratorFn = (Story) => (
   <MuiThemeProvider theme={theme}>
     <ThemeProvider theme={theme}>
       <CssBaseline />
-      <Story />
+      <div
```

Comment:
Temporary `style` hack because MUI's Typography chain breaks in this version of Storybook -- specifically for `fontFamily`. I believe it's an emotion confound that will be resolved once we upgrade SB.